README.md

```diff
@@ -1,2 +1,163 @@
-# ToDolistsm
-In this project, you will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.
+# To do List
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 TO DO list<a name="about-project"></a>
+
+**To do list ** is developed in HTML, CSS, JavaScript, and Es 6 modules,webback
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>
+      HTML <br>
+      CSS
+      </a>
+    </li>
+  </ul>
+</details>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+features
+- [Live Demo link](https://github.com/samuelkinyanjui/todolistsm)
+
+## Project Descriptive Video <a name="desc-video"></a>
+
+- [Project Descriptive Video link]()
+
+##
```

Comment:
- [ ] @samuelkinyanjui Great work on your documentation, however the live demo and description video link are not leading to the actual demo and project description, kindly consider checking this, incase you don't have the demo you can choose to delete this section.

Reply:
As suggested by the previous reviewer, kindly add `coming soon` for the live demo link and then remove the project description video since there is none at the moment 😄 . It is always recoommended to have link that takes you to the appropriate site